pkg/controller/context.go

```diff
@@ -18,65 +18,81 @@ import (
 	"context"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/gorilla/sessions"
 )
 
 // contextKey is a unique type to avoid clashing with other packages that use
 // context's to pass data.
 type contextKey string
 
-var (
-	// contextKeyAuthorizedApp is a context key used for the authorized app.
-	contextKeyAuthorizedApp = contextKey("authapp")
-
-	// contextKeyRealm is a context key for the realm.
-	contextKeyRealm = contextKey("realm")
-
-	// contextKeyTemplate is a context key for the template.
-	contextKeyTemplate = contextKey("template")
-
-	// contextKeyUser is a context key used for the user.
-	contextKeyUser = contextKey("user")
+const (
+	contextKeyAuthorizedApp = contextKey("authorizedApp")
+	contextKeyRealm         = contextKey("realm")
+	contextKeySession       = contextKey("session")
+	contextKeyTemplate      = contextKey("template")
+	contextKeyUser          = contextKey("user")
 )
 
-// WithAuthorizedApp sets the AuthorizedApp in the context.
-func WithAuthorizedApp(ctx context.Context, u *database.AuthorizedApp) context.Context {
-	return context.WithValue(ctx, contextKeyAuthorizedApp, u)
+// WithAuthorizedApp stores the authorized app on the context.
+func WithAuthorizedApp(ctx context.Context, app *database.AuthorizedApp) context.Context {
+	return context.WithValue(ctx, contextKeyAuthorizedApp, app)
 }
 
-// AuthorizedAppFromContext gets the currently logged in AuthorizedApp from the request context.
-// If no AuthorizedApp exists on the request context, or if the value in the request is
-// not a AuthorizedApp object, the result will be nil.
+// AuthorizedAppFromContext retrieves the authorized app from the context. If no
+// value exists, it returns nil.
 func AuthorizedAppFromContext(ctx context.Context) *database.AuthorizedApp {
 	v := ctx.Value(contextKeyAuthorizedApp)
 	if v == nil {
 		return nil
 	}
 
-	authorizedApp, ok := v.(*database.AuthorizedApp)
+	t, ok := v.(*database.AuthorizedApp)
 	if !ok {
 		return nil
 	}
-	return authorizedApp
+	return t
 }
 
-// WithRealm sets the realm in the cotnext.
-func WithRealm(ctx context.Context, r *database.Realm) context.Context {
-	return context.WithValue(ctx, contextKeyRealm, r)
+// WithRealm stores the current realm on the context.
+func WithRealm(ctx context.Context, app *database.Realm) context.Context {
```

Comment:
The variable r makes more sense here instead of app?

Reply:
You may have caught me copy-pasting 😄